src/pages/Flavors.test.js

```diff
@@ -1,36 +1,33 @@
+import { render, fireEvent } from '@testing-library/react';
 import React from 'react';
-import { Provider } from 'react-redux';
-import renderer from 'react-test-renderer';
-import configureStore from 'redux-mock-store';
-import { initialState as appInitialState } from 'reducers/application';
-import { initialState as flavorInitialState } from 'reducers/flavor';
-import { initialState as flavorsInitialState } from 'reducers/flavors';
-import ConnectedFlavors, { Flavors } from './Flavors';
-import { withMemoryRouter } from 'utils/testing';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { actions as flavorActions } from 'reducers/flavor';
+
+import PaginatedFlavors, { Flavors } from './Flavors';
+
+jest.mock('react-redux', () => {
+  const dispatch = jest.fn();
+
+  return {
+    ...jest.requireActual('react-redux'),
+    useDispatch: jest.fn(() => dispatch),
+    useSelector: jest.fn()
+  };
+});
 
 jest.mock('components/Pagination/Pagination', () => {
-  const pagination = require('utils/testing').mockComponent('Pagination');
-  const PagerInfo = require('utils/testing').mockComponent('PagerInfo');
+  const pagination = () => <div>Pagination</div>;
```

Comment:
`require('utils/testing').mockComponent('Pagination')` doesn't work right w/ `@testing-library/react`

Reply:
You should be able to use `pagination` lowercase, but that's a good point.